jax/_src/lax/lax.py

```diff
@@ -2571,38 +2567,35 @@ def _minmax_translation_rule(c, x, y, *, minmax=None, cmp=None):
 ad.defjvp_zero(lt_p)
 
 
-def _convert_element_type_shape_rule(operand, *, new_dtype, old_dtype):
+def _convert_element_type_shape_rule(operand, *, new_dtype):
```

Comment:
What would you think about using `dtype` rather than `new_dtype` as the argument name?

Reply:
That makes a ton of sense!

Reply:
Actually, because the argument to the lax function takes argument `new_dtype`, I think it might be better to leave the primitive parameter to have the same name.